pfhedge/features/container.py

```diff
@@ -124,7 +124,7 @@ def forward(self, input: Tensor) -> Tensor:
     def get(self, time_step: Optional[int]) -> Tensor:
         return self(self.inputs.get(time_step))
 
-    def of(self: TM, derivative=None, hedger=None) -> TM:
+    def of(self: TM, derivative: "Derivative" = None, hedger: "Hedger" = None) -> TM:  # type: ignore  # NOQA
```

Comment:
文字列を使わざるを得ないのであれば理由をコメントで書いておいて欲しいです

```suggestion
    def of(self: TM, derivative: Derivative = None, hedger: Hedger = None) -> TM:  # type: ignore  # NOQA
```

Reply:
よくよく見たら，この部分のsignatureがそもそも間違っていた(derivativeはNoneが許容されない)ので，親のclassのsignatureのtypeに一致させる修正をしました (後続別コミット)